### ISSUE TITLE
NPTR causing crash to desktop due to etching acid missing icon

### ISSUE THREAD

**Issue by LachimHeigrim** [MineMaarten/PneumaticCraft#841]
I reported details under #824 some times ago. I'd love to use etching acid in a Modtweaker custom recipe for Mariculture vat. Since Mariculture does not support 1.8, I'm stuck with 1.7.10 so quite interested in a fix.

I glanced through PneumaticCraft code and it seems etching acid uses different registration code than other liquids. I might attempt to fix it and send pull request, but a) Do you want me to mess with that code or is it different for a good reason and I better not touch it as a newcomer? b) Do you still publish builds or 1.7.10 ? and c) If I attempt which branch should I fork from to make merging to 1.7.10 easy or you? I don't see any useful branches/tags.

Cheers,
Lach

**Comment by Fayti1703** [comment]
[(Direct link to comment)](https://github.com/MineMaarten/PneumaticCraft/issues/824#issuecomment-202153622)

**Comment by AEnterprise** [comment]
that's a mariculture crash

**Comment by LachimHeigrim** [comment]
Yes, but reliably due to PnC not registering etching acid icon correctly, thus causing NPTE when Mariculture tries to retrieve it in it's render code.
